notebooks/official/model_monitoring/model_monitoring.ipynb

```diff
@@ -202,7 +202,6 @@
       "outputs": [],
```

Comment:
<p>This is a massive amount of code in a cell. I would recommend at least explaining (sentence or two) what each helper function is for.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/841/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>The helper functions are hidden because they are not germane to the educational goals of this lesson. Hiding the cell also compensates for there being a lot of code, i.e. you won't even see it unless you look for it. In the interest of avoiding making this notebook even longer than it already is, I'd prefer to leave it as is but if you feel strongly I'll add some text.</p>